plugins/out_opentelemetry/opentelemetry.h

```diff
@@ -59,6 +59,34 @@ struct opentelemetry_context {
     char *host;
     int port;
 
+    /* metadata keys */
+    flb_sds_t logs_observed_timestamp_metadata_key;
+    struct flb_record_accessor *ra_observed_timestamp_metadata;
+
+    flb_sds_t logs_timestamp_metadata_key;
+    struct flb_record_accessor *ra_timestamp_metadata;
+
+    flb_sds_t logs_severity_text_metadata_key;
+    struct flb_record_accessor *ra_severity_text_metadata;
+
+    flb_sds_t logs_severity_number_metadata_key;
+    struct flb_record_accessor *ra_severity_number_metadata;
+
+    flb_sds_t logs_trace_flags_metadata_key;
+    struct flb_record_accessor *ra_trace_flags_metadata;
+
+    flb_sds_t logs_span_id_metadata_key;
+    struct flb_record_accessor *ra_span_id_metadata;
+
+    flb_sds_t logs_trace_id_metadata_key;
+    struct flb_record_accessor *ra_trace_id_metadata;
+
+    flb_sds_t logs_attributes_metadata_key;
+    struct flb_record_accessor *ra_attributes_metadata;
+
+    flb_sds_t logs_instrumentation_scope_metadata_key;
+    flb_sds_t logs_resource_metadata_key;
```

Comment:
@nokute78 what do you intended to do with this property ?

There is a need for setting OTLP resource attributes https://github.com/fluent/fluent-bit/issues/7649